<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationPage</class>
 <widget class="QWidget" name="ConfigurationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>440</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>879</x>
     <y>30</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>60</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>120</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_4">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>180</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_5">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>150</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_6">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRed">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>200</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBlack">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>360</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWhite">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>360</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYellow">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>200</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBlue">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>280</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGreen">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>280</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="VideoBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>801</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>Video Stream</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHMax">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>H MAX</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHMin">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>H MIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSMax">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>S MAX</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSMin">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>S MIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVMax">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>V MAX</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVMin">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>180</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>V MIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPause">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>500</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
